File: springboot/open-api/src/main/java/com/openapi/domain/dto/ws/response/ControlCommandResponse.java
package com.openapi.domain.dto.ws.response;

import com.openapi.domain.dto.ws.base.CommonResultDto;
import lombok.Data;
import lombok.EqualsAndHashCode;
import java.util.Map;

@EqualsAndHashCode(callSuper = true)
@Data
public class ControlCommandResponse extends CommonResultDto {
    private String commandId;
    private String command;
    private Map<String, String> params;
}

File: app/android/app/src/main/java/com/magicvector/domain/dto/ws/response/ControlCommandResponse.kt
package com.magicvector.domain.dto.ws.response

import com.magicvector.domain.dto.ws.base.CommonResultDto
import java.util.*

data class ControlCommandResponse(
    val commandId: String,
    val command: String,
    val params: Map<String, String>
) : CommonResultDto()